pynestml/codegeneration/resources_nest/point_neuron/common/NeuronHeader.jinja2

```diff
@@ -382,22 +383,43 @@ private:
 {%- endif %}
 
 private:
-  {% if (neuron.get_multiple_receptors())|length > 1 -%}
+  {% if has_multiple_synapses -%}
   /**
    * Synapse types to connect to
-   * @note Excluded upper and lower bounds are defined as INF_, SUP_.
+   * @note Excluded lower and upper bounds are defined as MIN_, SUP_.
```

Comment:
I'm fine using either infimum and supremum or minimum and maximum, but please use either one or the other combination.